MCprep_addon/util.py

```diff
@@ -186,19 +186,19 @@ def obj_copy(base, context=None, vertex_groups=True, modifiers=True):
         
 
 @functools.cache
-def getBV():
+def getBV(major_version, minor_version):
 	if hasattr(bpy.app, "version"):
-		return bpy.app.version
+		return True if bpy.app.version >= (major_version, minor_version) else False
```

Comment:
Simplify for the same effect:

```
return bpy.app.version >= (major_version, minor_version)
```

Reply:
If changing the input to be a tuple, then this just becomes `return bpy.app.version >= version` where version is a tuple, noting that with python logic all of these are True statements:

```
(1, 2, 3) > (1, 2, 0)
(1, 3) > (1, 2, 3)
(2, 80) > (2, 79)
(3,) > (2, 79)  # Comma forces it to be treated as a tuple
(1, 0, 0) > (1, 0) # Interesting edge case to be aware of.
```